File: src/main/cs/00/Problem0005.cs
using System;
using System.Diagnostics;
using Euler.test.cs;

namespace Euler.main.cs._00
{
    public class Problem0005
    {
        public Problem0005()
        {
            var stopWatch = new Stopwatch();
            stopWatch.Start();
            ulong result = 1;

            // Solution:
            ulong[] primes = {3, 5, 7, 11, 13, 17, 19};
            foreach (var p in primes) result *= p * (20 / p);


            stopWatch.Stop();
            var elapsedTime = stopWatch.Elapsed.ToString();
            Console.WriteLine("Result:\t" + result + "\tTime:\t" +
                              (double.Parse(elapsedTime.Substring(elapsedTime.LastIndexOf(":") + 1, 2)) >= 1
                                  ? double.Parse(elapsedTime.Substring(elapsedTime.LastIndexOf(":") + 1)) + " s"
                                  : double.Parse(elapsedTime.Substring(elapsedTime.IndexOf(".") + 1)) / 10_000 +
                                    " ms"));
            if (Test.DoBenchmark)
                Benchmark.AddTime(5, double.Parse(elapsedTime.Substring(elapsedTime.IndexOf(".") + 1)) / 10_000);
        }
    }
}

File: src/main/java/de/swerik/euler/_00/Problem0005.java
package de.swerik.euler._00;

public class Problem0005 {
    public Problem0005() {
        long startTime = System.nanoTime();
        long result = 1;

//        outer:
//        for (int i = 20; ; i += 20) {
//            for (int j = 1; j < 21; j++) {
//                if (i % j != 0) {
//                    continue outer;
//                }
//            }
//            result = i;
//            break;
//        }

        int[] primes = {3, 5, 7, 11, 13, 17, 19};
        for (int p : primes) {
            result *= p * (20 / p);
        }

        long timeToResolve = System.nanoTime() - startTime;
        if (((double) timeToResolve / 1_000_000) >= 1000) {
            System.out.println("Result:\t" + result + "\tTime:\t" + ((double) timeToResolve / 1_000_000_000) + "s");
        } else {
            System.out.println("Result:\t" + result + "\tTime:\t" + ((double) timeToResolve / 1_000_000) + "ms");
        }
//        (int n=((double) timeToResolve / 1_000_000))>1000?n:n/1000
    }
}
